### PR TITLE
feat: persist use new image

### PR DESCRIPTION
**Pull request by bodinsamuel**
- Pin to Persist 
There is no latest in this new image, on purpose
- Lint markdown
- Clarify ENCRYPTION_KEY
- Increase version semver
Not sure it's done like that, let me know

### CODE REVIEW

**Review by TBonnin — APPROVED** [review]
I am not super familiar with the helm charts but lgtm

File: charts/nango/values.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ persist:
   name: persist
   url: http://nango-persist
   replicas: 1
+  tag: e6fd036d8bf2f5b7f9a8cd574746821e72b3801e # 2024/06/07
```
Comment (TBonnin) [inline charts/nango/values.yaml:48]: I wonder what kind of release strategy we should have to update this

Comment (bodinsamuel) [reply charts/nango/values.yaml:48]: I don't know either, I just think it's better to pin but yeah we need to do it regularly

Comment (khaliqgant) [reply charts/nango/values.yaml:48]: We could on new image update trigger an action in this repo which pushes a new tag version. I've used https://github.com/peter-evans/repository-dispatch with success in the past

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**